server.js

```diff
@@ -25,17 +25,17 @@ app.post("/public/server.js", (req, res) => {
 // console.log(directories);
 
 const openai = new OpenAI({
-    apiKey: process.env.OPENAI_API,
+    apiKey: process.env.openaiAPI,
 });
 
-app.get("/openai/:food", async (req, res) => {
```

Comment:
@josueJURE I removed the :food part of the open ai. As it's the wrong destination.